Resources/Prototypes/Recipes/Lathes/electronics.yml

```
@@ -777,4 +790,24 @@
+    Glass: 500
+    Gold: 100
+
+- type: latheRecipe # Goobstation - making tesla coils and radiation collectors craftable
+  id: RadiationCollectorCircuitboard
+  result: RadiationCollectorCircuitboard
+  category: Circuitry
+  completetime: 7
+  materials:
+    Steel: 100
+    Glass: 500
+    Gold: 100
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)